astropy/io/fits/tests/test_convenience.py

```diff
@@ -23,12 +23,12 @@ class TestConvenience(FitsTestCase):
     def test_resource_warning(self):
         warnings.simplefilter('always', ResourceWarning)
         with catch_warnings() as w:
-            data = fits.getdata(self.data('test0.fits'))
```

Comment:
I prefer assignment to `_` in these cases. Makes it clearer that the function actually returns something but we don't care.